data/mods/Aftershock/mobs/robots.json

```diff
@@ -319,8 +319,8 @@
     "id": "mon_utilibot_fire",
     "type": "MONSTER",
     "copy-from": "mon_utilibot",
-    "name": "firefighter robot",
-    "description": "One of the many models of fire-fighting robot formerly in use by local fire departments and emergency services.  Designed for entering burning buildings and other situations deemed too dangerous for human firefighters.",
+    "name": "TsKBEM Brandspojt",
+    "description": "Due to the significance placed on fire-fighting in the Soviet Union, it made perfect sense that with the advancements in robotics meant there were would be a machine designed for fire-fighting.  The Brandspojt was this, and its development was plagued by feature creep and development hell to such an extent that the government had to rein in the project.  It was eventually launched, and while it lacked the advanced features that it was advertised to include; it was effective at its job of both complimenting personnel and replacing them, enough so that several other countries scrambled to negiotiate for their own fire-fighting departments.",
```

Comment:
Significance placed on fire-fighting? (Some further copyediting is also needed, but I'd like to clarify this first.)